app.py

```diff
@@ -20,5 +21,28 @@ def Login():
             flash('Login Unsuccessful. Invalid Email/Password')
     return render_template('login.html',title='Login | SAC Portal, IIT Mandi',form=form)
 
+#Example  https://host/leaveclub?userID='B19188'&clubID='C10001'
+@app.route('/leaveclub', methods = ['GET','POST'])
```

Comment:
- [ ] Make this route, POST only

Reply:
It is showing method not allowed in POST, I thing it is GET, so I have make this route only GET

Reply:
it will show method not allowed if you access it through browsers address bar because it only sends get request. that is meant to happen. that is why you should test api with postman
![Screenshot from 2020-03-31 00-43-52](https://user-images.githubusercontent.com/45260243/77952216-e77e1380-72e8-11ea-98df-14f7de4f0fc2.png)

Reply:
Also here i have used query params but use request body to transfer data

Reply:
You have changed the route method, but you are still using query params for data tranfer